proto/sentry_protos/snuba/v1/endpoint_aggregate_bucket.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+enum Function {
+  FUNCTION_UNSPECIFIED = 0;
+  FUNCTION_SUM = 1;
+  FUNCTION_AVERAGE = 2;
+  FUNCTION_COUNT = 3;
+  FUNCTION_P50 = 4;
+  FUNCTION_P95 = 5;
+  FUNCTION_P99 = 6;
+  FUNCTION_AVG = 7;
+}
+
+message AggregateBucketRequest {
+  RequestMeta meta = 1;
+  TraceItemFilter filter = 2;
+  repeated AttributeAggregation aggregations = 3;
+  uint64 granularity_secs = 4;
+  repeated AttributeKey group_by = 5;
```

Comment:
Is there a way to limit the number of results the group by will cause? As well we'll likely need an orderby too

Reply:
Why would the query be ordered by anything other than time?